plugins/config/lib/configService.js

```diff
@@ -2264,6 +2337,21 @@ function getPluginConfiguration(identifier, pluginLocation, serverSettings,produ
 };
 exports.getPluginConfiguration = getPluginConfiguration;
 
+function getAllowedPlugins(scope, options, username) {
+  let policy = AGGREGATION_POLICY_MERGE
+  let identifier = "org.zowe.zlux.bootstrap"
+  let filename = "allowedPlugins.json"
+  let relativePath = identifier + "/plugins"
+  let serverSettings = options.serverConfig
+  let productCode = "ZLUX"
+  let pluginLocation = "C:\\Users\\skeerthy\\post90\\zlux\\zlux-app-manager\\bootstrap"
```

Comment:
It's hard to get the plugin location from here, so I suggest moving this entire function out to webapp.js
Here's how it can work:
webapp has: id='org.zowe.zlux.bootstrap', filename='allowedPlugins.json', serverConfig.... but most importantly the location of org.zowe.zlux.bootstrap is found at https://github.com/zowe/zlux-server-framework/blob/12488f38c01ac051ee052dcf27101fd63aa8b25e/lib/webapp.js#L221
`plugins: function(plugins, options){`
The "plugins" argument is an array that contains objects which have `location` as the full path to the directory they are in.
`pluginDefs` doesnt, because it is intentionally stripped.